tests/_logger_nm.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python-wrapper.sh
+#!/usr/bin/env python3
```

Comment:
I prefer to completely drop `/usr/bin/env`.

Reply:
I reverted the shebangs for tests to previous state. The /usr/bin/env python-wrapper.sh line has been kept as it is needed to run the tests in distcheck target